lib/bootloader_pvm.pm

```diff
@@ -162,10 +162,46 @@ sub prepare_pvm_installation {
     wait_still_screen;
 }
 
+sub assert_cmd {
+    my ($cmd, $timeout) = @_;
+    # clear screen with \033c as it seems to be running on restricted ksh and ott
+    type_string "echo -e '\\033c'\n";
+    type_string "$cmd; echo -e \"\\033c\\ncmd: $cmd\\nret: \$\?\\n\"\n";
+    assert_screen "cmd-successful", $timeout;
+}
+
+sub load_iso {
+    my $self = shift;
+    # avoid iso filename collision
+    my $iso_dir        = get_required_var('ISO_DIR');
+    my $vios_iso_dir   = get_required_var('_SECRET_VIOS_ISO_DIR');
+    my $iso            = get_required_var('ISO');
+    my $iso_url        = get_required_var('ISO_URL');
+    my $vtopt_name     = get_required_var('VTOPT_NAME');
+    my $vios_disk_name = get_required_var('VIOS_DISK_NAME');
+    my $vios_password  = get_required_var('_SECRET_VIOS_PASSWORD');
+    my $vios_username  = get_required_var('_SECRET_VIOS_USERNAME');
+    my $vios_hostname  = get_required_var('_SECRET_VIOS_HOSTNAME');
+    if (check_var('UPSTREAM_ISO', 'yes')) {
+        $iso_url = get_var('UPSTREAM_ISO_URL');
+    }
```

Comment:
As you are adding new variables, please add them to [os-autoinst-distri-opensuse/variables.md](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md)

Reply:
Yes, I have added it to variables.md .You can find the new variables info here ,https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19066/files#diff-eb176a275e164bf44779feb951ab9bbcbb60959e2b5227a4e23101962d69299e